### PR TITLE
Add an internalVersion number to compare against the send version

### PR DESCRIPTION
**Pull request by MorrisJobke**
* use the "latest" number for displaying and download URLs
* current NC 10.0.0 sends 9.1.0.16 as version ;)
* adjusted config.php accordingly

This properly works here with my instance.

cc @LukasReschke @nickvergessen

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -111,8 +116,12 @@
 		],
 	],
 	'daily' => [
+		'9.1' => [
+			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable10.zip',
+			'web' => 'https://docs.nextcloud.org/server/9/admin_manual/maintenance/manual_upgrade.html',
+		],
 		'9.0' => [
-			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest.zip',
+			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable9.zip',
```
Comment (MorrisJobke) [inline config/config.php]: Ah ... I forgot that little 💎 ... we most likely don't want our users to run master ... stable branches should be fine here ;)

Comment (nickvergessen) [inline config/config.php]: latest is the highest latest-stableX, btw

**Comment by nickvergessen** [comment]
tests fail

**Comment by MorrisJobke** [comment]
> tests fail

Fixed - I forgot the dailies.

File: config/config.php (hunk 1 above)
Comment (MorrisJobke) [inline config/config.php]: Really? @LukasReschke said that latest.zip ist the master based archive.

Comment (nickvergessen) [inline config/config.php]: https://gist.github.com/nickvergessen/e78eac9c84cc0e0d91d50723f3e43166#file-latest-php-L71

Comment (nickvergessen) [inline config/config.php]: anyway in that case it would be stable10 anyway, so the change is fine

Comment (LukasReschke) [inline config/config.php]: 🙈 

… @nickvergessen is right it seems 🙈

**Comment by MorrisJobke** [comment]
@nickvergessen @LukasReschke Good to merge?

**Comment by nickvergessen** [comment]
Fine by me :+1:

**Pull request merged**